linux_os/guide/system/software/gnome/gnome_screen_locking/dconf_gnome_screensaver_lock_enabled/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: fedora,ol7,ol8,rhel7,rhel8
+prodtype: fedora,ol7,ol8,rhel7,rhel8,sle12
```

Comment:
At this point we don't have a sle-15 stig for, this item, but in case one drops we did not want to mess with turning if = sle12 to if in [sle12,sle15], we had too many places where we needed to do that because we just didn't think far enough ahead when we did sle-12.